docs/source/api/symbolic/predefined/find.rst

```diff
@@ -0,0 +1,7 @@
+find
+====
+
+This module contains the find function.
```

Comment:
Make this a little more descriptive, maybe?

Reply:
sure